tigershark/tools/utils.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
This module happens to include `print` statements that are also parseable as valid `print()` function calls. It seems reasonable to include it, so py2 and py3 behave the same.